rdt/transformers/numerical.py

```diff
@@ -250,8 +250,10 @@ def __init__(self, missing_value_replacement=None, model_missing_values=False,
             learn_rounding_scheme=learn_rounding_scheme,
             enforce_min_max_values=enforce_min_max_values
         )
-        self._distributions = self._get_distributions()
 
+        self.distribution = distribution  # Distribution initialized by the user
```

Comment:
Were we just not using this at all previously? Are we actually using it now? If not should we just dump the parameter?